src/test/scala/org/zalando/zhewbacca/IAMClientSpec.scala

```diff
@@ -3,21 +3,23 @@ package org.zalando.zhewbacca
 import akka.actor.ActorSystem
 import org.specs2.mutable.Specification
 import org.zalando.zhewbacca.metrics.NoOpPlugableMetrics
-import play.api.http.Port
+import play.api.http.{DefaultFileMimeTypes, FileMimeTypesConfiguration, Port}
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.libs.ws.WSClient
-import play.api.mvc.{Action, Handler, Result, Results}
+import play.api.mvc._
 import play.api.test.WsTestClient
 import play.api.{Application, Configuration, Mode}
 import play.core.server.Server
 
 import scala.concurrent.duration._
 import scala.concurrent.{Await, ExecutionContext}
 
-// @see https://www.playframework.com/documentation/2.5.x/ScalaTestingWebServiceClients
+// @see https://www.playframework.com/documentation/2.6.x/ScalaTestingWebServiceClients
 class IAMClientSpec extends Specification {
   sequential
 
+  implicit val mimeTypes = new DefaultFileMimeTypes(new FileMimeTypesConfiguration())
```

Comment:
This is needed for Results.Ok.sendResource. It would be better to inject mime types but with the current test setup I couldn't find a proper way to do it this way.